backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/process/C_BPartner_Update_Memo.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2024 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.bpartner.process;
+
+import de.metas.bpartner.BPartnerId;
+import de.metas.bpartner.service.IBPartnerDAO;
+import de.metas.process.JavaProcess;
+import de.metas.process.Param;
+import de.metas.util.Services;
+import org.compiere.model.I_C_BPartner;
+
+public class C_BPartner_Update_Memo extends JavaProcess
+{
+	private final IBPartnerDAO bpartnerDAO = Services.get(IBPartnerDAO.class);
+
+	@Param(parameterName = "Memo")
+	private String p_Memo;
+
+	@Override
+	protected String doIt() throws Exception
+	{
+		final I_C_BPartner bPartner = bpartnerDAO.getById(BPartnerId.ofRepoId(getRecord_ID()));
+		bPartner.setMemo(p_Memo);
+		bpartnerDAO.save(bPartner);
+		return "@Success@";
+	}
```

Comment:
Write a method in `de.metas.bpartner.service.IBPartnerBL` that sets the Memo field and save the partner.
Somehting like `de.metas.bpartner.service.impl.BPartnerBL.updateMemo((@NonNull final BPartnerId bpartnerId), @NonNull String memo)`